projects/wmf_default.yaml

```diff
@@ -99,6 +99,28 @@ paths:
             #purge:
             #  host: '239.128.0.112'
             #  port: 4827
+
+            # A list of pages that we don't currently want to re-render on
+            # each edit. Most of these are huge bot-edited pages, which are
+            # rarely viewed in any case. 
+            rerenderBlacklist:
+              www.wikidata.org:
+                Q21558717: true
+                Q21505108: true
+                Q21481867: true
+                Q21521425: true
+              en.wikipedia.org:
+                Wikipedia:Administrators'_noticeboard/Incidents: true
+                User:JamesR/AdminStats: true
+                User:Kudpung/Dashboard: true
+              ur.wikipedia.org:
+                نام_مقامات_ایل: true
+                نام_مقامات_ڈی: true
+                نام_مقامات_جے: true
+                نام_مقامات_جی: true
+                نام_مقامات_ایچ: true
```

Comment:
Also this title .. https://ur.wikipedia.org/w/index.php?title=%D9%86%D8%A7%D9%85_%D9%85%D9%82%D8%A7%D9%85%D8%A7%D8%AA_%D8%A7%DB%8C%D9%85&action=history